### PR TITLE
fix(orgx) add uniorg-parse and uniorg-rehype to deps

### PR DESCRIPTION
**Pull request by venikx**
Fixes: #78
Depends on: #79

### CODE REVIEW

File: packages/orgx/package.json [hunk 1]
```diff
@@ -34,6 +34,8 @@
     "hast-util-to-estree": "^2.2.1",
     "periscopic": "^3.0.0",
     "unified": "^10.0.0",
+    "uniorg-parse": "workspace:^",
```
Comment (venikx) [inline packages/orgx/package.json:37]: @rasendubi I'm not familiar with pnpm, is this meant to be like that when adding packages from the workspace?

Comment (rasendubi) [reply packages/orgx/package.json:37]: hm... I'm actually not sure. We'll likely need specific versions when we publish the packages (users of `orgx` through npm won't know what "workspace:^" is)

**Comment by rasendubi** [comment]
It looks like we need to update the lock file:
image

File: packages/orgx/package.json (hunk 1 above)
Comment (rasendubi) [reply packages/orgx/package.json:37]: ok, let's merge this and see if publishing works 😅

Comment (rasendubi) [reply packages/orgx/package.json:37]: yay, that worked! the published package.json has these versions fixed (instead of "workspace:^")

**Comment by rasendubi** [comment]
Thanks for the bug report and the fix!

If you want to get your hands dirty with uniorg, you could also look at https://github.com/rasendubi/uniorg/issues/74 — that's another easy bug and it might also affect you

File: packages/orgx/package.json (hunk 1 above)
Comment (venikx) [reply packages/orgx/package.json:37]: Talking about yolo'ing it :D

**Comment by venikx** [comment]
> Thanks for the bug report and the fix!
> 
> If you want to get your hands dirty with uniorg, you could also look at #74 — that's another easy bug and it might also affect you

I'll most likely run into that at some point yeah, I'll take a look at it when it trips me up.

**Pull request merged**